e2e/test/scenarios/admin/databases/actions.cy.spec.js

```diff
@@ -10,7 +12,7 @@ describe(
         restore(`${dialect}-writable`);
         cy.signInAsAdmin();
 
-        cy.request(`/api/database/${WRITABLE_DB_ID}`).then(({ body }) => {
+        visitDatabase(WRITABLE_DB_ID).then(({ response: { body } }) => {
```

Comment:
@iethree please let me know if this change makes sense to you.

Unless there was some other reason behind it, there's no reason to "manually" send a request because we visit that database anyway and can assert on the response.

One more hint: please avoid using `cy.request` without a method. It falls back to `GET` but it's not guaranteed it's going to work between Cypress versions.